housingbayarea/bloom
# fix: hcv community updates

This PR addresses #4113


## Description

This PR pulls over much of the work of https://github.com/metrotranscom/doorway/pull/662 but with the copy of the ticket above.
 
## How Can This Be Tested/Reviewed?

On localhost:3100, the script runner endpoint passing an id of the jurisdiction you want to add the community type too.

Log in to the partner site and add the new community type to a listing
Go to the public site and make sure it comes through properly (and is translated correctly)

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
